app/schemas/document.py

```diff
@@ -275,6 +275,7 @@ class SectionWrapper(BaseModel):
             TableBlockNode,
             SubsectionNode,
             EquationNode,
+            EquationInlineNode,
```

Comment:
Is this correct, though? Should it be possible that an inline equation is a child of a section wrapper I was under the impression that only block-level nodes should be direct children of a section. 

Inline equations should be treated like other inline nodes, such as bold or italic, and should only be accepted as children of paragraphs. 

(It's also possible that I don't have a full understanding of how the nesting of these blocks work)

Reply:
You're absolutely correct, thanks for catching that!